opentelemetry-sdk/src/export/logs/mod.rs

```diff
@@ -91,6 +92,25 @@ pub trait LogExporter: Send + Sync + Debug {
         // By default, all logs are enabled
         true
     }
+
+    /// This is a hint to ensure that the export of any Spans the exporter
+    /// has received prior to the call to this function SHOULD be completed
+    /// as soon as possible, preferably before returning from this method.
+    ///
+    /// This function SHOULD provide a way to let the caller know
+    /// whether it succeeded, failed or timed out.
+    ///
+    /// This function SHOULD only be called in cases where it is absolutely necessary,
+    /// such as when using some FaaS providers that may suspend the process after
+    /// an invocation, but before the exporter exports the completed spans.
+    ///
+    /// This function SHOULD complete or abort within some timeout. This function can be
+    /// implemented as a blocking API or an asynchronous API which notifies the caller via
+    /// a callback or an event. OpenTelemetry client authors can decide if they want to
+    /// make the flush timeout configurable.
+    fn force_flush(&mut self) -> BoxFuture<'static, ExportResult> {
```

Comment:
should we instead have `async fn force_flush(&mut self) -> ExportResult` - it's more cleaner, without boxing overhead

Reply:
+1

Reply:
Updated it to be async.